rogii/build_common.cmake

```diff
@@ -0,0 +1,113 @@
+if(
+    NOT DEFINED ROOT
+    OR NOT DEFINED ARCH
+)
+    message(
+        FATAL_ERROR
+        "Assert: ROOT = ${ROOT}; ARCH = ${ARCH}"
+    )
+endif()
+
+set(
+    BUILD
+    0
+)
+
+if(DEFINED ENV{BUILD_NUMBER})
+    set(
+        BUILD
+        $ENV{BUILD_NUMBER}
+    )
+endif()
+
+set(
+    TAG
+    ""
+)
+
+if(DEFINED ENV{TAG})
+    set(
+        TAG
+        "$ENV{TAG}"
+    )
+else()
+    find_package(
+        Git
+    )
+
+    if(Git_FOUND)
+        execute_process(
+            COMMAND
+                ${GIT_EXECUTABLE} rev-parse --short HEAD
+            OUTPUT_VARIABLE
+                TAG
+            OUTPUT_STRIP_TRAILING_WHITESPACE
+        )
+        set(
+            TAG
+            "_${TAG}"
+        )
+    endif()
+endif()
+
+include(
+    "${CMAKE_CURRENT_LIST_DIR}/version.cmake"
+)
+
+set(
+	ARCH
+	"anycpu"
+)
+
+set(
+    PACKAGE_NAME
+    "rapidjson-${ROGII_RAPIDJSON_VERSION}-${ARCH}-${BUILD}${TAG}"
+)
+
+
+set(
+    CMAKE_INSTALL_PREFIX
+    ${ROOT}/${PACKAGE_NAME}
+)
+
+set(
+    DEBUG_PATH
+    "${CMAKE_CURRENT_LIST_DIR}/../build/debug_${ARCH}"
```

Comment:
Please describe the problem you are facing with. (This is different from the previous implementation.)

Reply:
rename path to build folder and variable to BUILD_PATH